src/panels/gitConfig.html

```diff
@@ -1,4 +1,7 @@
 <div id="container-div">
+    <div class="detailsDiv">
+        <vscode-dropdown position="below" id="workspace-root-list-dropdown"> </vscode-dropdown>
+    </div>
```

Comment:
We could hide this if there is only one workspace. We can leave it as future enhancement

Reply:
Left it for now